data/transport missions.txt

```diff
@@ -1748,7 +1748,8 @@ mission "Terraforming 3"
 				`	"That is a stupid idea, and I want nothing to do with it."`
 					decline
 				`	"Well, if you think it will work, I'm willing to try it!"`
-			`	"Great!" says Amy. "I'll mark the asteroid on your radar. You just need to board it, install a small ion thruster we will give you, and return here."`
+			`	"Great!" says Amy. "I'll mark the asteroid on your radar. You just need to board it, install a small ion thruster, and return here."
```

Comment:
It's outside the scope of this PR. "Ion" you're talking about was there originally.